vip-helpers/vip-utils.php

```diff
@@ -73,6 +73,9 @@ function wpcom_vip_noncdn_uri( $path ) {
  * @return string HTML
  */
 function vip_powered_wpcom_img_html( $image ) {
+	//Stores constant to allow cache breaking if images need replaced
+	$IMG_VERSION = "20220317";
```

Comment:
should be:

```$img_version = '20220317'```
according to the coding standards of the project.

Change this and the other use of the variable and it should pass linting.

Reply:
Thanks Jesse. Updated both variables and changed the comment.